src/main/java/org/codice/jsonrpc/DocMethod.java

```diff
@@ -0,0 +1,19 @@
+package org.codice.jsonrpc;
+
+import java.util.Map;
+
+public class DocMethod implements Method {
+  private final Method method;
```

Comment:
I had to go look around for what a `Method` was. Then i found it and realized oh right its just a Function. 

feelsbadman.jpg

This type is not overrused and i'd still say this is more straightforward just leaving it as `Function<Map<String, Object>, Object>`.

I mean this is just a ✏️ probably i dont' really mind if its left as `Method`. It *is* nice and terse. Just meh. If the definition of the alias was more localized id love it way more 😂

Reply:
okay lets leave it and we'll see how it settles 😌